packages/application-config/src/substitute-variable-placeholders.ts

```diff
@@ -0,0 +1,154 @@
+import fs from 'fs';
```

Comment:
The file got renamed, I'll point out only the new things in this file. The rest is the same as before.